pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
```

Comment:
Note: the existence of a `pyproject.toml` could make build tooling angry depending on what generation of python build tools you're using. I can remove this if it messes with the library building and publication.

Reply:
Probably worth leaving this out for now. @MattiasMTS is looking at improving the dependencies/build tooling in #215